typescript-test-samples/async-architectures/src/tests/integration/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "async-architectures",
+  "version": "1.0.0",
+  "description": "async architectures testing example for NodeJS",
+  "main": "app.js",
+  "repository": "https://github.com/awslabs/aws-sam-cli/tree/develop/samcli/local/init/templates/cookiecutter-aws-sam-hello-nodejs",
+  "author": "SAM CLI",
+  "license": "MIT",
+  "dependencies": {
+    "@aws-sdk/client-s3": "^3.301.0",
+    "@aws-sdk/client-dynamodb": "^3.301.0",
+    "@aws-sdk/lib-dynamodb": "^3.301.0"
+  },
+  "scripts": {
+    "compile": "tsc",
+    "lint": "eslint '*.ts' --quiet --fix",
+    "test:unit": "jest --selectProjects unit",
+    "test:integration": "jest --selectProjects integration"
+  },
+  "devDependencies": {
+    "@types/aws-lambda": "^8.10.92",
+    "@types/jest": "^29.5.0",
+    "@types/node": "^18.11.4",
+    "@types/uuid": "^9.0.1",
+    "@typescript-eslint/eslint-plugin": "^5.10.2",
+    "@typescript-eslint/parser": "^5.10.2",
+    "esbuild": "^0.14.14",
```

Comment:
move to `dependencies` as commented on template file.

Reply:
Fixed.